### PR TITLE
updated about page MIT staff

### LINKED ISSUE #810: Update staff on the about page
Correct staff photos and info

#### Acceptance Criteria:

- [ ] Update the photo for Elizabeth DeRienzo with the one [here](https://github.mit.edu/storage/user/2678/files/7cc5da00-09c2-11ed-8f23-e52224b04c40)
- [ ]  remove Joe Martis
- [ ] add Randall Carter, with the image from https://openlearning.mit.edu/about/our-team/randall-carter
- [ ] change Curt Newton's title from "Publication Director" to simply "Director"

### PR DESCRIPTION
**Pull request by Wassaf-Shahzad**
#### Pre-Flight checklist

- [x] Screenshots and design review for any changes that affect layout or styling
  - [x] Desktop screenshots
  - [ ] Mobile width screenshots
- [ ] Testing
  - [ ] Code is tested
  - [x] Changes have been manually tested

#### What are the relevant tickets?
Close #810 

#### What's this PR do?
Updated staff info on about page.

#### How should this be manually tested?
- Start ocw-www
- Visit about page
- Verify all changes mention in issue #810 have been completed

#### Any background context you want to provide?
- The image for Randall Cater is of the wrong aspect ration and can be fixed with stretching but the results are not promising.
- Screenshot of current iteration is attached below
#### Screenshots (if appropriate)
Screenshot 2022-08-10 at 5 33 02 PM

### CODE REVIEW

File: www/content/staff/curt-newton.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 first_name: Curt
 last_name: Newton
-job_title: OCW Publication Director
```
Comment (MAbdurrehman12) [inline www/content/staff/curt-newton.md:4]: > change Curt Newton's title from "Publication Director" to simply "Director"

Currently it's `OCW Publication Director` so should it be changed to only `Director` or `OCW Director`? 
@pdpinch

**Review by MAbdurrehman12 — APPROVED** [review]
LGTM 👍

I also noticed this right/left blank space around Randall Cater's image, which you have already mentioned in the PR description. 
Replacing it with one having better dimensions and quality would be ideal.

File: www/content/staff/curt-newton.md (hunk 1 above)
Comment (pdpinch) [reply www/content/staff/curt-newton.md:4]: Good question. I don't know why some titles have "OCW" in front of them, when most do it. Let's remove it.

**Comment by Wassaf-Shahzad** [comment]
@pdpinch I merged you PR, Should I merge this too ?

**Review by pdpinch — APPROVED** [review]

**Review by pdpinch — APPROVED** [review]
Looks good to me, and it's also been approved by the OCW team, who reviewed the netlify version.

**Pull request merged**